source/documentation/using_the_api/register_components.md

```diff
@@ -12,21 +12,23 @@ Entries contain metadata about what and when data has changed.
 
 Entries contain the fields `entry-number`,  `entry-timestamp`, `item-hash` and `key`. 
 
-The `entry-number` is unique and defines an entry’s position within the ordered list of a register. The `entry-timestamp` is the time when an entry was introduced to a given register, but it is no guarantee of the order of entries in a register. The `entry-number` is.
+The entry number is unique and defines an entry’s position within the ordered list of a register. The entry timestamp is the time when an entry was introduced to a given register, but it is no guarantee of the order of entries in a register. The entry number is.
 
-Entries are connected to items by a given `item-hash`. 
+Each entry is connected to an item by a given item hash. Entries reference items in the `item-hash` field using the hash derived from the corresponding item's content.
 
-Each `key` is an alphanumeric string unique to the data in a register. The primary key (i.e. the unique identifier) always has the same name as the name of the register. For example, every entry in the `local-authority-eng` register has a field called `local-authority-eng`. Foreign keys, which are not always present, link to other registers. 
+Each key is an alphanumeric string unique to the data in a register. The primary key (i.e. the unique identifier) always has the same name as the name of the register. For example, every entry in the `local-authority-eng` register has a field called `local-authority-eng`. Foreign keys, which are not always present, link to other registers. 
```

Comment:
After reading the linking section in #77 I think there is no need to mention foreign keys here.